scripts/patch_lovyangfx_c5.py

```
@@ -0,0 +1,191 @@
+    for rel, subs in EDITS:
+        path = os.path.join(LIB_DIR, rel)
+        try:
+            with open(path) as f:
+                content = f.read()
+        except OSError as e:
+            fail("marker present but cannot read %s: %s" % (rel, e))
+        for _old, new, expected in subs:
+            if content.count(new) < expected:
+                fail("marker present but %s lacks the patched text %r — "
+                     "stale marker; wipe .pio/libdeps/%s/LovyanGFX and rebuild"
+                     % (rel, new[:60], env.subst("$PIOENV")))
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)